cmd/geth/main.go

```diff
@@ -101,7 +101,7 @@ var (
 		utils.UltraLightFractionFlag,
 		utils.UltraLightOnlyAnnounceFlag,
 		utils.LightNoSyncServeFlag,
-		utils.WhitelistFlag,
+		utils.AllowListFlag,
```

Comment:
```suggestion
		utils.WhitelistFlag,
		utils.AllowListFlag,
```
should it work or should we use only Whitelist flag?